src/tasks/07_groupByType.js

```diff
@@ -21,3 +21,16 @@
  */
 
 // <-- тут реализация groupByType
+export const groupByType = (array) => {
+    let result = {
+        legal: [],
+        physical: []
+    }
+    array.map(item => {
+        if (item.type === 'legal')
+            result.legal.push(Object.fromEntries(new Map([Object.entries(item)[0]]))) //item)
+        else result.physical.push(Object.fromEntries(new Map([Object.entries(item)[0]])))
+        return item;
```

Comment:
Так выглядело бы недостаточно круто?)
```suggestion
        if (item.type === 'legal')
            result.legal.push({ name: item.name }) //item)
        else result.physical.push({ name: item.name })
        return item;
```